lib/snapshot/snapshot.dart

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
```

Comment:
This whole file should be dead. libraries.json now decides what goes into platform.dill.

Reply:
Perhaps we coud open another PR for removing these - I can do that if you like